client/js/lounge.js

```diff
@@ -1269,45 +1269,65 @@ $(function() {
 		forms.find(".username").val(nick);
 	});
 
-	Mousetrap.bind([
-		"command+up",
-		"command+down",
-		"ctrl+up",
-		"ctrl+down"
-	], function(e, keys) {
-		var channels = sidebar.find(".chan");
-		var index = channels.index(channels.filter(".active"));
-		var direction = keys.split("+").pop();
-		switch (direction) {
-		case "up":
-			// Loop
-			var upTarget = (channels.length + (index - 1 + channels.length)) % channels.length;
-			channels.eq(upTarget).click();
-			break;
+	(function HotkeysScope() {
+		Mousetrap.bind([
+			"command+up",
+			"command+down",
+			"ctrl+up",
+			"ctrl+down"
+		], function(e, keys) {
+			var channels = sidebar.find(".chan");
+			var index = channels.index(channels.filter(".active"));
+			var direction = keys.split("+").pop();
+			switch (direction) {
+			case "up":
+				// Loop
+				var upTarget = (channels.length + (index - 1 + channels.length)) % channels.length;
+				channels.eq(upTarget).click();
+				break;
+
+			case "down":
+				// Loop
+				var downTarget = (channels.length + (index + 1 + channels.length)) % channels.length;
+				channels.eq(downTarget).click();
+				break;
+			}
+		});
 
-		case "down":
-			// Loop
-			var downTarget = (channels.length + (index + 1 + channels.length)) % channels.length;
-			channels.eq(downTarget).click();
-			break;
-		}
-	});
+		Mousetrap.bind([
+			"command+k",
+			"ctrl+shift+l"
+		], function(e) {
+			if (e.target === input[0]) {
+				clear();
+				e.preventDefault();
+			}
+		});
 
-	Mousetrap.bind([
-		"command+k",
-		"ctrl+shift+l"
-	], function(e) {
-		if (e.target === input[0]) {
-			clear();
-			e.preventDefault();
-		}
-	});
+		Mousetrap.bind([
+			"escape"
+		], function() {
+			contextMenuContainer.hide();
+		});
 
-	Mousetrap.bind([
-		"escape"
-	], function() {
-		contextMenuContainer.hide();
-	});
+		var colorsHotkeys = {
+			k: "\x03",
```

Comment:
Is there any reason not to simply bind <kbd>Ctrl</kbd>+<kbd>K</kbd> on every platform? That's what a number of existing clients use, and it would reduce The Lounge's learning curve if that keybinding remained the same.